content/articles/beginner's-guide-to-lottie-animations-in-android-using-kotlin/index.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: beginner's-guide-to-lottie-animations-in-android-using-kotlin
+title: Beginner's guide to Lottie animations in Android using Kotlin 
+description: This article explains how to use Lottie which is an open source animation library in Android applications using Kotlin. Lottie is beautiful and helps you achieve a clean UI, therefore achieving a good user experience.
+author: carol-musyoka
+date: 2021-05-31T00:00:00-10:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/beginner's-guide-to-lottie-animations-in-android-using-kotlin/hero.jpg
+    alt: Beginner's guide to Lottie animations in Android using Kotlin 
+---
+
+### Introduction
+Designing animations can be tedious and technical for most developers. **Lottie** is an open-source animation library developed by *Airbnb*. It renders Adobe After Effects with BodyMovin. **BodyMovin** is an Adobe After Effects plugin that exports animations to a JSON file. Lottie makes it easy to work with animations without downloading heavy animation files.
+Lottie helps developers achieve a clean UI, therefore achieving a good user experience. Lottie is easy to use, and the files can be accessed on their [site](https://lottiefiles.com/).
+Lottie is supported on mobile (Android, iOS, and React Native), web, and desktop applications.
+
+### Why use Lottie
+- Simple UI elements - Lottie is light and the JSON files exported are small-sized. 
+- Dynamic scenes - You can tell a narrative across several shots, making it interactive.
+- Alter animations to your liking - The animations offered on Lottie can be altered from speed to color to progress hence meeting the developer's need.
+- Improved user experience.
+- Pairs well across all mobile devices(in our case), even slow-performing ones, since it is lightweight.
+- The animations also have a clear resolution.
+
+This article will share how to implement Lottie animations in onboarding screens using Kotlin. Since the project is a bit wide, we shall only go through all the basics of Lottie animations on the app. 
+
+### Prerequisites
+
+Basic knowledge of android and Kotlin.
+
+### Goal
+At the end of this tutorial, the reader should understand how to use and implement Lottie animations in Android applications. 
+
+### Step one - Create a new Android Studio Project.
+In this step, we will create a new Android Create a new android project on Android Studio
+
+### Step two - Choosing an animation file to use on your app
+
+Head over to Lottie's [website](www.lottiefiles.com) and choose the animations of your choice. Most of the animations are free, while others one has to pay to use them.  Download the .json file. In our project, let us create a res/raw folder. Save the .json animations on there.
```

Comment:
You can explain how the raw folder is created using arrows like this: This is an example explaining how to create a new fragment. "Select New → Fragment → Fragment(Blank)."